python/CMakeLists.txt

```diff
@@ -35,17 +35,25 @@ project(
 
 ################################################################################
 # - User Options  --------------------------------------------------------------
+option(CUML_UNIVERSAL "Build all cuML Python components." ON)
+option(CUML_CPU "Build only cuML CPU Python components." OFF)
+option(CUML_GPU "Build only cuML GPU Python components. Not supported yet, defaulting to CUML_UNIVERSAL" OFF)
```

Comment:
If this isn't actually supported yet I would either 1) comment out the option and set the variable directly `set(CUML_GPU ${CUML_UNIVERSAL})` or at minimum 2) [mark this option as advanced](https://cmake.org/cmake/help/latest/command/mark_as_advanced.html). The former is preferable IMO.